src/config/bridging.ts

```diff
@@ -0,0 +1,33 @@
+import { ARBITRUM, AVALANCHE } from "./chains";
+
+const SOLANA_BRIDGING_IDS = {
+  [ARBITRUM]: 23,
+  [AVALANCHE]: 6,
+};
+
+type BridgingOptionsReturn =
+  | [
+      {
+        name: string;
+        generateLink: (chainId: number) => string;
+      }
+    ]
+  | undefined;
+
+const BRIDGING_OPTIONS = {
```

Comment:
would be good to specify type here to increase intellisense

BRIDGING_OPTIONS: {[symbol: string]: BridgingOptions[]}